`TestRevisionNumberStored` failure on main recently

**Describe the bug**

```
--- FAIL: TestRevisionNumberStored (0.00s)
    main_test.go:55: Git revision could not be determined - got '' but expected to match regular expression '^[0-9a-f](40)$'
        Did you specify `-ldflags "-X github.com/taskcluster/taskcluster/v55/workers/generic-worker.revision=<GIT REVISION>"` in your go test command?
        Try building generic-worker using the /workers/generic-worker/build.(sh|cmd) script in the taskcluster monorepo.
```

**To Reproduce**
To match what CI is doing:

```
cd workers/generic-worker
export GW_TESTS_RUN_AS_CURRENT_USER=true
export GORACE=history_size=7
export CGO_ENABLED=1

go install -tags multiuser -v -ldflags '-X main.revision=8b8c2f92f348dd125177f719e4def3e280d9a34c' ./...
go test -tags multiuser -failfast -timeout 20m -ldflags '-X github.com/taskcluster/taskcluster/v54/workers/generic-worker.revision=8b8c2f92f348dd125177f719e4def3e280d9a34c' -race -vet=off . -run TestRevisionNumberStored
```

--- a/changelog/issue-6513.md
+++ b/changelog/issue-6513.md
@@ -1,0 +1,4 @@
+audience: developers
+level: silent
+reference: issue 6513
+---

--- a/infrastructure/tooling/src/release/tasks.js
+++ b/infrastructure/tooling/src/release/tasks.js
@@ -201,6 +201,7 @@ module.exports = ({ tasks, cmdOptions, credentials }) => {
         'clients/client-go/**',
         'clients/client-shell/**',
         'internal/**',
+        'taskcluster/ci/generic-worker/**',
         'tools/**',
         'ui/docs/reference/workers/websocktunnel.mdx',
         // Provide explicit list of allowed file extensions so that

--- a/taskcluster/ci/generic-worker/kind.yml
+++ b/taskcluster/ci/generic-worker/kind.yml
@@ -167,9 +167,9 @@ tasks:
             cp "${{TASK_USER_CREDENTIALS}}" next-task-user.json
             # IMPORTANT - run go test with GW_TESTS_RUN_AS_CURRENT_USER=true *before* running it without
             # otherwise tests that call `go run ....` will write go object files to .cache as root
-            GW_TESTS_RUN_AS_CURRENT_USER=true GORACE=history_size=7 CGO_ENABLED=1 go test -tags "${{ENGINE}}" -failfast -timeout 20m -ldflags "-X github.com/taskcluster/taskcluster/v54/workers/generic-worker.revision=${{GITHUB_SHA}}" ${{RACE}} ${{VET}} .
+            GW_TESTS_RUN_AS_CURRENT_USER=true GORACE=history_size=7 CGO_ENABLED=1 go test -tags "${{ENGINE}}" -failfast -timeout 20m -ldflags "-X github.com/taskcluster/taskcluster/v55/workers/generic-worker.revision=${{GITHUB_SHA}}" ${{RACE}} ${{VET}} .
           fi
-          GORACE=history_size=7 CGO_ENABLED=${{CGO_ENABLED_TESTS}} go test -tags "${{ENGINE}}" -failfast -timeout 20m -ldflags "-X github.com/taskcluster/taskcluster/v54/workers/generic-worker.revision=${{GITHUB_SHA}}" ${{RACE}} ${{VET}} ./...
+          GORACE=history_size=7 CGO_ENABLED=${{CGO_ENABLED_TESTS}} go test -tags "${{ENGINE}}" -failfast -timeout 20m -ldflags "-X github.com/taskcluster/taskcluster/v55/workers/generic-worker.revision=${{GITHUB_SHA}}" ${{RACE}} ${{VET}} ./...
           # The next line is edited by infrastructure/tooling/src/generate/generators/golangci-lint-version.js
           #   DO NOT CHANGE HERE!
           ../../../golangci-lint/golangci-lint-$GOLANGCI_LINT_VERSION-*/golangci-lint run --build-tags "${{ENGINE}}" --timeout=5m
@@ -311,9 +311,9 @@ tasks:
         - set GORACE=history_size=7
         - copy "%TASK_USER_CREDENTIALS%" "%CD%\next-task-user.json"
         - set GW_TESTS_RUN_AS_CURRENT_USER=true
-        - go test -tags "%ENGINE%" -failfast -timeout 20m -ldflags "-X github.com/taskcluster/taskcluster/v54/workers/generic-worker.revision=%GITHUB_SHA%" %RACE% %VET% .
+        - go test -tags "%ENGINE%" -failfast -timeout 20m -ldflags "-X github.com/taskcluster/taskcluster/v55/workers/generic-worker.revision=%GITHUB_SHA%" %RACE% %VET% .
         - set GW_TESTS_RUN_AS_CURRENT_USER=
-        - go test -tags "%ENGINE%" -failfast -timeout 20m -ldflags "-X github.com/taskcluster/taskcluster/v54/workers/generic-worker.revision=%GITHUB_SHA%" %RACE% %VET% ./...
+        - go test -tags "%ENGINE%" -failfast -timeout 20m -ldflags "-X github.com/taskcluster/taskcluster/v55/workers/generic-worker.revision=%GITHUB_SHA%" %RACE% %VET% ./...
         - |
           :: assumption here is that if something inside the if fails, we'll get a non zero exit code
           :: i've also made it an if/else so that one of them has to run, as there should always be a

